### ISSUE TITLE
The H1 is identical on every page, regardless of content due to Site Title block level

### ISSUE THREAD

**Issue by jessynd** [WordPress/twentytwentyfour#54]
## Issue summary

The Site Title in the navigation bar or header template of the theme is wrapped in an `<h1>` element. This causes the H1 of every single page to be the same regardless of the content of the page. The first level heading on the page should be the **page title,** not the site title.
<img width="526" alt="Screen Shot 2023-08-24 at 4 01 54 PM" src="https://github.com/WordPress/twentytwentyfour/assets/5340550/6733f23e-06e5-4f3f-998c-78fd3afec9ce">

## Current code
```HTML
<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}}} -->
<div class="wp-block-group"><!-- wp:site-title /--></div>
<!-- /wp:group --></div>
<!-- /wp:group -->
```

## Steps to reproduce

1. Open up any page in your WordPress site with the TwentyTwentyFour theme
2. Inspect the site title in the left hand side of the header block
3. Observe the h1 wrapped around the site title.

## Recommendation

1. The site title block allows us to select the heading level used when outputted to the page. In this case, the level should be set to paragraph or level: 0. This will output the site title as a paragraph instead of a heading and it can then be styled according to the design spec. 
2. Amend each of the page templates so that the top level heading where the page title is outputted is an H1 and not an H2. 

### Code for files header.html, header-copy.html, header-writer.html, header-portfolio.html

```html
<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}}} -->
<div class="wp-block-group"><!-- wp:site-title {"level":0} /--></div>
<!-- /wp:group --></div>
<!-- /wp:group -->
```

## Specifications
**Relevant Templates**
[header-copy.html](https://github.com/WordPress/twentytwentyfour/blob/trunk/parts/header-copy.html)
[header-portfolio.html](https://github.com/WordPress/twentytwentyfour/blob/trunk/parts/header-portfolio.html)
[header-writer.html](https://github.com/WordPress/twentytwentyfour/blob/trunk/parts/header-writer.html)
[header.html](https://github.com/WordPress/twentytwentyfour/blob/trunk/parts/header.html)

WCAG Principle: Perceivable
WCAG SC: 1.3.1  Info and Relationships

**Comment by lefty55104** [comment]
The site title is also an H1 in the footer. The recommended steps above would then also apply to:

- footer-copy.html
- footer-portfolio.html
- footer-writer.html
- footer.html

**Comment by jessynd** [comment]
I gave fixing this a shot in my [PR](https://github.com/WordPress/twentytwentyfour/pull/59)

**Comment by carolinan** [comment]
I re-opened this to keep the context for the following issue:

> The first level heading on the page should be the page title, not the site title.

With the site title changed from H1 to paragraph, many templates are missing an H1.

**Comment by MaggieCabrera** [comment]
> I re-opened this to keep the context for the following issue:
> 
> > The first level heading on the page should be the page title, not the site title.
> 
> With the site title changed from H1 to paragraph, many templates are missing an H1.

Can we close this one now, @carolinan ?

**Comment by carolinan** [comment]
No, all templates are not merged yet.
Somewhere, the H1 on the home template was removed, so instead of two, there is now 0 😆

**Comment by carolinan** [comment]
Related, I received feedback from @afercia that the H3 headings in the default footer (about, privacy, social media) should be H2.
Otherwise they look like subheadings below "Join 900+ subscribers"

**Comment by afercia** [comment]
> Related, I received feedback from @afercia that the H3 headings in the default footer (about, privacy, social media) should be H2.

Rationale:

<img width="409" alt="Screenshot 2023-09-20 at 14 38 44" src="https://github.com/WordPress/twentytwentyfour/assets/1682452/0fff7701-553e-4219-a007-7bfce5ae3f69">

A heading level is skipped.
Scenario:
- Users add some content with H2 headings.
- The last heading text is: `About Carolina` 🙂 
- At this point, the headings `About`, `Privacy`, and `Social media` would be semantically and hierarchically sub-headings of `About Carolina`, as if they identified content sections that are children of `About Carolina`, which is clearly incorrect.

**Comment by carolinan** [comment]
I found out that using the class `screen-reader-text` wont work because it also hides the H1 heading in the editors.

**Comment by luminuu** [comment]
How are we going to deal with the missing `h1`? Adding it to the pattern means we're going to add it everywhere where the pattern is inserted, so more than one h1 could appear on every page/post.

**Comment by MaggieCabrera** [comment]
I think we can duplicate the markup for this template. Let's have the home page pattern use h1 and the partial pattern use h2.

**Comment by luminuu** [comment]
@carolinan has added some missing h1's in this PR today: #563, so I think we're good now with this.
